api/src/main/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandler.java

```diff
@@ -268,4 +269,55 @@ public Vehicle getLatestMotTest(TradeServiceRequest request, Context context) th
             throw new InternalServerErrorException(e, context.getAwsRequestId());
         }
     }
+
+    /**
+     * Get the latest vehicle information and MOT test recorded for the vehicle specified by MOT Test Number.
+     *
+     * @param request Describes the request. For this method, we require only the registration as a path parameter.
+     * @param context AWS Lambda request context
+     * @return The vehicle and its latest MOT, if found.
+     * @throws TradeException If there is a retrieval error or the vehicle and test are not found.
+     */
+    public Vehicle getLatestMotTestByMotTestNumber(TradeServiceRequest request, Context context) throws TradeException {
+
+        try {
+            logger.trace("Entering getLatestMotTestByMotTestNumber");
+            if (request.getPathParams().getNumber() != null) {
```

Comment:
I chose to keep it consistent with the rest of the methods (which don't create a variable), however I agree a variable should be created. Should I keep it consistent or change it?